packages/snap/src/keyring.ts

```diff
@@ -194,14 +199,22 @@ export class SimpleKeyring implements Keyring {
     return walletMatch;
   }
 
-  #generateKeyPair(): {
+  #getKeyPair(pk?: string): {
```

Comment:
The existing code used `pk` for private key, but it can be confusing since `pk` is usually used to abbreviate _public key_. For private keys we usually use `sk`.

Do you mind renaming this? You can switch to `sk` or `privateKey`. Thanks!

Reply:
Good point. Changed from `pk` to `privateKey` in [30eba50](https://github.com/MetaMask/snap-simple-keyring/pull/58/commits/30eba50648f87a85e40e5dd32c5bd24f78bafc18)